DocuSignSamplePlugin/DocuSignPlugin/pom.xml

```diff
@@ -42,6 +43,14 @@
       <version>6.0</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>jace</groupId>
+      <artifactId>jace</artifactId>
+      <version>3.0.11</version>
+      <scope>system</scope>
```

Comment:
Isn't system scope deprecated? If so, it will stop working when they remove it.
The readme file needs to be updated with instructions to copy the libraries from the ICN installation to the lib folder.

Reply:
The system scope works for now. Another option is to update the readme file to install the JAR files to the local Maven repository so the compile scope can be used.
https://maven.apache.org/plugins/maven-install-plugin/examples/generic-pom-generation.html

Reply:
Updated!